tests/functional/vault/test_misc.py

```diff
@@ -48,8 +48,8 @@ def test_reject_ether(gov, vault):
                 1,
             ],
         ),
-        ("deposit", [1]),
-        ("withdraw", [1]),
+        ("deposit", []),
+        ("withdraw", []),
```

Comment:
for test completeness, we should have all the different possibilities

Reply:
Great point!

Reply:
Will add additional cases!